webapp/src/app/shared/phaser/tilemap/tile-drawer.ts

```diff
@@ -358,6 +366,34 @@ export class TileDrawer extends Phaser.Plugin {
 		});
 	}
 	
+	private renderPreviewById() {
```

Comment:
This method contains some duplicated code. Maybe modify the renderPreview function to cover both use cases or just extract the duplicated code